app/templates/logout.html

```diff
@@ -1,4 +1,4 @@
 {% extends "base.html" %}
 {% block content %}
-<h3>Will see soon</h3>
+<div class="alert alert-success logout" role="alert">You've been logged out successfully.</div>
```

Comment:
I think `.` (dot) is extra here. Think would be better `!`
Also think `You've been` is extra. Maybe `Logged out successfully` is enough